mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -12,62 +12,83 @@
     data-unload-msg="{% trans "You are leaving this page, but have ticked payments. &#10;&#10;You must click &#8216;I&#8217;ve credited payments in NOMIS&#8217; if payments have been credited." %}">
     {% csrf_token %}
 
-    <div class="totals js-StickyHeader"  id="tour-step-total">
-      <div class="container js-RunningTotal" data-label="{% trans 'Total payments you have credited: ' %}">
-        <p>
-          {% blocktrans with total=total|currency highlight="<strong>&pound;" highlightend="</strong>" %}
-            Control total: {{ highlight }}{{ total }}{{ highlightend }}
-          {% endblocktrans %}
-        </p>
-      </div>
-    </div>
 
     {% if object_list %}
-    {{ form.errors }}
-
-    {% include "cashbook/includes/batch_buttons.html" %}
+      {% if form.errors or form.non_field_errors %}
```

Comment:
Yep, I wasn't suggesting doing it right now, more a case of making a note against this change to make @maxf aware that it might need to be done in the future. 

*(Probably looked like I was just critiquing my own work)*